sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -73,10 +73,19 @@ extensions:
                          -e containerized=true      \
                          -e deployment_type=origin  \
                          /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml
+        # install atomic rpm
+        sudo yum install atomic -y
+        # only origin, openvswitch and node docker images are run as system containers
+        sudo atomic pull --storage ostree docker:openshift/origin:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/openvswitch:$( cat ./ORIGIN_TAG )
+        sudo atomic pull --storage ostree docker:openshift/node:$( cat ./ORIGIN_TAG )
         ansible-playbook -vv --become               \
                          --become-user root         \
                          --connection local         \
                          --inventory sjb/inventory/ \
+                         -e openshift_use_system_containers=true \
+                         -e openshift_docker_systemcontainer_image_registry_override=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7 \
```

Comment:
I did: https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible_extended_conformance_install_system_containers/29/console. Running another one to verify the last changes: https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible_extended_conformance_install_system_containers/30/console